aten/src/ATen/CMakeLists.txt

```diff
@@ -149,41 +149,46 @@ endif(MKLDNN_FOUND)
 list(APPEND ATen_CPU_DEPENDENCY_LIBS cpuinfo)
 
 if(NOT MSVC AND NOT EMSCRIPTEN)
-  # Preserve values for the main build
-  set(__aten_sleef_build_shared_libs ${BUILD_SHARED_LIBS})
-  set(__aten_sleef_build_tests ${BUILD_TESTS})
-
-  # Unset our restrictive C++ flags here and reset them later.
-  # Remove this once we use proper target_compile_options.
-  set(OLD_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
-  set(CMAKE_CXX_FLAGS)
-
-  # Bump up optimization level for sleef to -O1, since at -O0 the compiler
-  # excessively spills intermediate vector registers to the stack
-  # and makes things run impossibly slowly
-  set(OLD_CMAKE_C_FLAGS_DEBUG ${CMAKE_C_FLAGS_DEBUG})
-  IF(${CMAKE_C_FLAGS_DEBUG} MATCHES "-O0")
-    string(REGEX REPLACE "-O0" "-O1" CMAKE_C_FLAGS_DEBUG ${OLD_CMAKE_C_FLAGS_DEBUG})
-  ELSE()
-    set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O1")
-  ENDIF()
+  hunter_add_package(sleef)
+  find_package(sleef CONFIG REQUIRED)
+  list(APPEND ATen_CPU_DEPENDENCY_LIBS sleef::sleef)
+  if(NOT HUNTER_ENABLED)
```

Comment:
That will be the source of conflicts in the future. For the sake of creating a patch with minimal changes, I think we can ignore indentation and do:

```cmake
if(NOT HUNTER_ENABLED)

set(__aten_sleef_build_shared_libs ${BUILD_SHARED_LIBS}) # <<<<< same indentation as in the original code
set(__aten_sleef_build_tests ${BUILD_TESTS})
...

endif()
```